zomes/remote_signal/coordinator/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "remote_signal_coordinator"
+name = "remote_signal"
```

Comment:
```suggestion
name = "remote_signals"
```

Reply:
Heh... I'm willing to try : ) but part of the problem of this pr has been collisions with naming, so this lack of plural may be making it work at the moment.

Reply:
Indeed:

```
error: two packages named `remote_signals` in this workspace:
- /home/neonphog/projects/wind-tunnel/scenarios/remote_signals/Cargo.toml
- /home/neonphog/projects/wind-tunnel/zomes/remote_signal/coordinator/Cargo.toml
```

Reverting this.